packages/builder/test/helpers/project.ts

```diff
@@ -54,3 +60,22 @@ export function useFixtureProjectCopy(srcProjectName: string) {
 
   after(() => fsExtra.removeSync(projectPath));
 }
+
+/**
+ * Allows tests to interact with a clean fixture project.
+ * Allows to inspect the output file after running the test by cleaning before running.
+ */
+export function useCleanFixtureProject(projectName: string) {
+  useFixtureProject(projectName);
+  useEnvironment((algobEnv: AlgobRuntimeEnv) => {
+    return algobEnv.run(TASK_CLEAN, {});
+  });
+
+  beforeEach(function () {
+    try {
+      fs.unlinkSync(testFixtureOutputFile)
+    } catch (err) {
+      // ignored
+    }
+  })
+}
```

Comment:
Isn't this a copy-paste from some task test? (eg TASK_CLEAN)? If yes, then let's remove the function form that test.

Reply:
I moved this to here from deploy tests.